src/main/java/com/cloudbees/jenkins/plugins/BitBucketTrigger.java

```diff
@@ -164,7 +151,7 @@ public String getLog() throws IOException {
          * Writes the annotated log to the given output.
          */
         public void writeLogTo(XMLOutput out) throws IOException {
-            new AnnotatedLargeText<BitBucketWebHookPollingAction>(getLogFile(), Charset.defaultCharset(),true,this).writeHtmlTo(0,out.asWriter());
+            new AnnotatedLargeText<BitBucketWebHookPollingAction>(getLogFile(), Charset.defaultCharset(), true, this).writeHtmlTo(0, out.asWriter());
```

Comment:
Unrelated change to the PR - even if it is wrong I prefer not to modify this and correct it on a separate PR